src/main/java/ro/unibuc/hello/entities/Policy.java

```diff
@@ -29,4 +30,18 @@ public String toString() {
     public PolicyDTO toDTO() {
         return new PolicyDTO(id, name, statements);
     }
+
+    public int hasRightsToAction(ActionDTO action){
+        String actionCode = action.getCode();
+        boolean hasRights;
+        for(var statement : statements) {
+            for(var statementAction : statement.getActions()) {
+                if(statementAction.equals(actionCode)) {
+                    hasRights = statement.getEffect().equals(EffectTypes.ALLOW);
+                    return hasRights ? 1 : -1;
```

Comment:
Coding the results of a function like this is generally considered bad practice, it would be better if you define an Enum that has the possible results (enum AccessRightTypes {DENY, ALLOW, NEUTRAL}) and return one of those instead of -1, 1 and 0. Another option is to use the already existent EffectTypes enum but think of a way to handle the 0 result. (I recommend the first option though)